src/Storefront/Pagelet/Footer/FooterPagelet.php

```diff
@@ -30,4 +37,20 @@ public function getServiceMenu(): ?CategoryCollection
     {
         return $this->serviceMenu;
     }
+
+    /**
+     * @deprecated tag:v6.7.0 - reason:return-type-change - Will only return CategoryCollection
```

Comment:
`CategoryCollection` should be `PaymentMethodCollection` right? And I guess next one also is wrong

Reply:
whoopsy, copy paste issue :sweat_smile: